laguinho/routes/datasets.py

```diff
@@ -11,6 +11,14 @@ def publish():
     datasets_metadata.append(result)
     return jsonify(result), 201
 
-@datasets.route("/datasets", methods=['GET'])
+@datasets.route("/datasets/", methods=['GET'])
 def get_datasets():
     return jsonify(datasets_metadata)
+
+@datasets.route("/datasets/<name>", methods=['GET'])
+def get_datasets_by_name(name):
+  search_result = list(
+    filter(lambda dataset: dataset['name'] == name, datasets_metadata))
+  
+  result = search_result[0] if len(search_result) > 0 else None
+  return jsonify({"dataset": result}), 200
```

Comment:
A gente vai ter mais alguma coisa além do dataset nesse JSON? porque se não eu não vejo muito problema em retornar um jsonify(result) diretão mesmo 🤔

Reply:
Até pra padronizar com os outros endpoints

Reply:
Eu tinha colocado assim pra facilitar a verificação, se o `dataset` for nulo, quer dizer que não encontrou nenhuma ocorrência

Reply:
Quando não encontra um dataset é melhor retornar um 404 com um json de erro, eu acho